ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@mattklein123 Apologies, I misread your previous comment - I wrongly inferred that you were talking about the base glibc image itself. While it will be best effort in that the base image (frolvlad/alpine-glibc) may or may not be updated regularly with the latest glibc, what would be an obstacle for CI to put out an alpine image? I'm asking because I see the image as something the community would most probably use a lot in their deployments. Considering that the glibc release cadence is rather low, this won't create too many tags over time either.